src/main/daml/Daml/Finance/Data/Time/DateClock.daml

```diff
@@ -55,13 +55,14 @@ template DateClock
           Unit currentDate = date
           clock = this with date = Unit $ addDays currentDate 1
         clockCid <- create clock
-        eventCid <- create DateClockUpdateEvent with
-          providers
-          observers
-          id = eventId
-          description = eventDescription
-          eventTime = toUTCTime this
-          date = currentDate
+        eventCid <- create DateClockUpdateEvent
+          with
```

Comment:
often in the code, we do 
```
create Foo with 
  a = 1
  b = 2
```  
instead of having a separate line for `with`. maybe something for the style guide?

Reply:
It's consistent with the style guide for whenever we have the `with` keyword. You are correct that we don't have something explicit in the style guide for `create`.